### ISSUE TITLE
Incorrect solution for fizz and buzz

### ISSUE THREAD

**Issue by KhalilKhunji** [ISRAA-ASHOOR/js-arrays-lab#2]
https://github.com/ISRAA-ASHOOR/js-arrays-lab/blob/44710acf9b8fdef4e51c45bdf117fe4e18b48085/app.js#L71-L79

While your result for fizzbuzz was correct, fizz and buzz were not.

This is because you chained the if statements with else ifs; so that if during the loop a condition was fulfilled, it would only push the number into one of the arrays, when in fact you should be pushing them into any number of arrays so long as the condition for being part of that array is fulfilled.

In other words, this should be something like:

```
nums.forEach(num => {
    if (num % 3 === 0 && num % 5 === 0) {
        fizzbuzz.push(num); 
    } 
    if (num % 5 === 0) {
        buzz.push(num); 
    } 
    if (num % 3 === 0) {
        fizz.push(num); 
    }
});
```

Can you see the difference?
